Dockerfile

```diff
@@ -62,7 +62,7 @@ WORKDIR $GEOIPS_PACKAGES_DIR
 
 RUN apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y \
-        imagemagick wget libsqlite3-0 libtiff5 libcurl4 libcurl3-gnutls libgeos-3.9.0 libgeos-dev \
+        wget libsqlite3-0 libtiff5 libcurl4 libcurl3-gnutls libgeos-3.9.0 libgeos-dev \
```

Comment:
Ahh, is this what prompted you to try to get the dockerfile working? I am fine with either just deleting this file altogether, or leaving this change to be merged, and fix up the dockerfile in the other issue/branch/PR

Reply:
Pretty sure we no longer need a lot of those libraries also

Reply:
Yes! I went to run it and realized it wouldn't build with/without it.